src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PersonLink } from '../components/Person/PersonLink';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const { slug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setHasError(false);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const parentInfo = (parentName: string) => {
+    const parent = people.find(person => person.name === parentName);
+
+    if (parent) {
+      return (
+        <PersonLink person={parent} />
+      );
+    }
+
+    return parentName;
+  };
```

Comment:
If it is a function, its name should start with a verb. If it is a component, the name must be capitalized.
It is also worth taking it outside the current component, so that it is not created anew during each rendering.